openmindat/countries.py

```diff
@@ -92,13 +96,40 @@ def save(self, FILE_NAME = ''):
                 None
 
             Example:
-                >>> cr = countriesRetriever()
-                >>> cr.save()
+                >>> cr = CountriesListRetriever()
+                >>> cr.page(2).save()
         '''
         file_name = FILE_NAME
         
         self.saveto('', file_name)
         
+    def get_object(self):
+        '''
+        Executes the query to retrieve the country data as a dictionary.
+
+        Returns:
+            list of dictionaries, or a dictionary if "page" parameter is included.
+
+        Example:
+            >>> cr = CountriesListRetriever()
+            >>> france = cr.page(2).get_object()
+
+        '''
+        
+        print("Retrieving countries. This may take a while... ")
+       
+        params = self._params
+        end_point = self.end_point
+        
+        ma = mindat_api.MindatApi()        
+        #clears params for next get statement     
+        self._init_params()
+
+        if "page" in params:
```

Comment:
if "page" in params:
    return [ma.get_mindat_dict(params, end_point)]
else:
    return ma.get_mindat_list_object(params, end_point)

Reply:
Wrapping the dict object with square parenthesis converts it to a list with one dict element. Therefore, the data structures will become identical for unified usage.